gdal/frmts/grib/degrib18/g2clib-1.0.4/pngunpack.c

```diff
@@ -55,7 +59,8 @@ g2int pngunpack(unsigned char *cpack,g2int len,g2int *idrstmpl,g2int ndpts,
       if (nbits != 0) {
 
          ifld=(g2int *)calloc(ndpts,sizeof(g2int));
-         ctemp=(unsigned char *)calloc(ndpts*4,1);
+	 int nbytes = nbits/8;
```

Comment:
As this is a .c file, variable should be declared before first statement, otherwise will not compile on Windows